frontend-vue/src/stores/domains.ts

```diff
@@ -172,6 +172,18 @@ export const useDomainsStore = defineStore("domains", () => {
     return domainLevel !== undefined && domainLevel > 0;
   };
 
+  const allBoundedContextsNames = computed(() => {
```

Comment:
I would prefer to use the existing boundedContexts store for `allBoundedContextsNames`

https://github.com/trustbit/Contexture/blob/main/frontend-vue/src/stores/boundedContexts.ts#L20